core/server/web/shared/middlewares/image/handle-image-sizes.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+const sharp = require('sharp');
+const storage = require('../../../../adapters/storage');
+const activeTheme = require('../../../../services/themes/active');
+
+const SIZE_PATH_REGEX = /^\/size\/([^/]+)\//;
```

Comment:
Would be nice to add a sample of what exactly the regex is meant to match or a unit test that shows `req.url = /size/123hello` not going through but `/size/123hello/` does